dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -76,6 +114,8 @@ printf "%s\n" "Operating System Environment"
 printf "%s\n" "-----------------------------"
 printf "%23s: %s\n" "Operating System" "$(get_os_info)"
 printf "%23s: %s\n" "Kernel" "$(get_kernel_info)"
+printf "%23s: %s\n" "System shell" "$(get_sys_shell)"
+printf "%23s: %s\n" "make's SHELL" "$(get_make_shell)"
```

Comment:
why `System shell` and `make's SHELL` (all capitals)?

Reply:
Im guessing because of this https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html